src/modal/modal.js

```diff
@@ -130,36 +130,32 @@ angular.module('ui.bootstrap.modal', ['ui.bootstrap.multiMap', 'ui.bootstrap.sta
               }).start().then(done);
             });
           }
+          // Notify {@link $modalStack} that modal is rendered.
+          var modal = $modalStack.getTop();
+          if (modal) {
+            $modalStack.modalRendered(modal.key);
```

Comment:
I want to change logic for rendered. I created Issue #6368. Please check it. I can't change position for modal window because DOM is not rendered while `opened` is firing.